wagtail/wagtailimages/models.py

```diff
@@ -441,16 +442,28 @@ def url(self):
 
     @property
     def attrs(self):
-        return mark_safe(
-            'src="%s" width="%d" height="%d" alt="%s"' % (escape(self.url), self.width, self.height, escape(self.image.title))
-        )
+        """
+        The src, width, height, and alt attributes for an <img> tag, as a HTML
+        string
+        """
+        return flatatt(self.attrs_dict)
```

Comment:
This is not exactly equivalent as `flatatt()` prepends a leading space.

> The returned string will contain a leading space followed by key="value", XML-style pairs.

That said I can't foresee that being an issue other than potentially increasing payload size of a few web pages by a byte.